src/holdingpen/components/ExceptionsTable.jsx

```diff
@@ -0,0 +1,170 @@
+import React, {Component} from 'react';
+import { Table, Input, Button, Icon } from 'antd';
+
+class exceptionsTable extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      filteredInfo: null,
+      data: props.exceptions,
+      record: {
```

Comment:
nested `state` is not good,  most likely a sing that you should separate those into smaller components.